checks/mixins/container_engine.py

```diff
@@ -67,3 +71,19 @@ def create_env_file(self):
     @run_before('run')
     def set_container_engine_env_launcher_options(self):
         self.job.launcher.options += [f'--environment={self.env_file}']
+
+
+class ContainerEngineCPEMixin(rfm.RegressionMixin):
+    @run_after('setup')
+    def set_container_mounts(self):
+       current_environ = self.current_environ
+       self.build_locally = False
+       if 'cpe_ce_image' in current_environ.resources:
```

Comment:
I didn't try it, but I think if there is no resources in the env it will just be ignored. 🤔 I think you don't need this check.